ArkhamOverlay/Data/CardInfo.cs

```diff
@@ -18,6 +18,7 @@ public class CardInfo : IHasImageButton {
 
         public CardInfo(ArkhamDbCard arkhamDbCard, int count, bool isPlayerCard, bool cardBack = false, bool isBonded = false) {
             Code = arkhamDbCard.Code;
```

Comment:
ImageId and Code are getting a little mixed here. Is there a reason to keep both?

Reply:
I'm not sure- Code is the actualy code in ArkhamDB- I originally was modifying it like I was using ImageID, but that felt dirty, so I created ImageID. We aren't using Code for anything now, so we could remove it, but it does mean something, so not sure if we should remove it or not.

Reply:
Is there ever a case where they would be different? If not, then I'd suggest, consolidating under one name.

Reply:
Yes, they are different when the image is the back of the card- then I append "Back" to the code to generate the image ID.